test/splunk/instrumentation/rails_test.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "rails"
+
+require "test_helper"
+require "opentelemetry/sdk"
+require "opentelemetry/instrumentation/rack"
+require "opentelemetry/instrumentation/action_pack"
+require "splunk/otel"
+require "splunk/otel/instrumentation/rack"
```

Comment:
```ruby
require "splunk/otel/<nested things>"
```
These probably shouldn't be here unless we are going to say they are optional somehow.